.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Ideally, we use `pnp` for this, but that seemed to be breaking things and I don't want to spend time of fixing it